lib/gollum-lib/markups.rb

```diff
@@ -59,5 +65,8 @@ class Markup
              :extensions => ['mediawiki','wiki'], :reverse_links => true)
     register(:pod, "Pod",
              :enabled => MarkupRegisterUtils::executable_exists?("perl"))
+    register(:bib, "BibTeX", :extensions => ['bib'],
+             :enabled => MarkupRegisterUtils::gems_exist?(["bibtex-ruby", "citeproc-ruby", "csl-styles"]),
+             :skip_filters => Proc.new {|filter| ![:YAML,:BibTeX,:Sanitize].include?(filter)})
```

Comment:
Never mind, github was being a bit ambiguous, but it's now clear that this refers to line 70. :)

What about: `Proc.new {|filter| true unless [:YAML,:BibTeX,:Sanitize].include?(filter)}`